docs/components/switch.md

```diff
@@ -51,12 +51,30 @@ Will place label text to the right of the switch. Can also be [slotted](#slots).
 
 <KInputSwitch v-model="labelPropChecked" :label="labelPropChecked ? 'on' : 'off'" />
 
+### disabled
+
+You can add `disabled` to the input to disallow interactivity.
+
+- `disabled`
+
 ```vue
 <KInputSwitch v-model="checked" label="disabled" disabled />
 ```
 
 <KInputSwitch v-model="labelPropChecked" label="disabled" disabled />
 
+### disabledTooltipText
+
+You can specify tooltip text to be displayed when the switch is disabled.
+
+- `disabledTooltipText`
+
+```vue
+<KInputSwitch v-model="checked" label="disabled" disabled disabledTooltipText="I'm disabled!" />
```

Comment:
When I viewed this code example on the page, it gets cutoff - could we format it so that it's on a few lines like this:

```
<KInputSwitch
  v-model="checked"
  label="disabled"
  disabled
  disabledTooltipText="I'm disabled!"
/>
```

![Screen Shot 2021-06-04 at 4 14 51 PM](https://user-images.githubusercontent.com/2080476/120871578-d8c86780-c550-11eb-884a-90cf5b9bfbc6.png)

Reply:
Good catch! I didn't even notice.

Fixed.

![image](https://user-images.githubusercontent.com/67973710/121051182-3e9e3480-c787-11eb-9ce1-4d28d2609c96.png)